lib/layout/data_grid.dart

```diff
@@ -474,6 +474,54 @@ class DataGridState extends WidgetState<DataGrid>
     setState(() {});
   }
 
+  void customSort(List dataList, String? sortKey, bool isAscendingOrder) {
```

Comment:
extract this to a utility and add unit tests. This is not UI logic and easily testable.

Reply:
Done